src/moj/components/ticket-panel/_ticket-panel.scss

```diff
@@ -0,0 +1,36 @@
+/* ==========================================================================
+   #TICKET PANEL
+   ========================================================================== */
+
+.moj-ticket-panel {
+   display: block;
+   flex-wrap: wrap;
+
+   @include govuk-media-query($from: desktop) {
+      display: flex;
+      flex-wrap: nowrap;
+
+      & > * + * {
+        margin-left: govuk-spacing(3);
+      }
+   }
+
+   &__content {
+      display: block;
+      position: relative;
+      background-color: govuk-colour("light-grey");
+      padding: govuk-spacing(3);
+      margin-bottom: govuk-spacing(3);
+      border-left: 5px solid $govuk-brand-colour;
+
+      &--red { border-left: 5px solid govuk-colour("red"); }
```

Comment:
Is this the right way to handle colour options? Should we have something like passing in a colour or some sort of override to make this more flexible. We should look at other components and the way they handle this.

Reply:
@johnnolan to look at options and feed back.

Reply:
After discussion with @gregtyler we concluded that this is the best way for now as it gives us a manageable set of options. We could explore dynamically looping the colour options and populating it, however this introduces complexity that would be better managed by contributors in PRs to add modifiers as needed.